x/mongo/driver/operation.go

```diff
@@ -557,6 +557,12 @@ func (op Operation) retryable(desc description.Server) bool {
 	return false
 }
 
+// SupportsRetryWrites returns true if retryable writes are supported which requires
+// that logicalSessionTimeoutMinutes exists, and server type is not standalone
+func SupportsRetryWrites(desc description.Server) bool {
```

Comment:
this function could go on description.Server

Reply:
done